docs/source/quickstart.rst

```diff
@@ -153,23 +153,41 @@ When you run the example, it outputs an MLflow run ID for that experiment. If yo
 ``mlflow ui``, you will also see that the run saved a ``model`` folder containing an ``MLmodel``
 description file and a pickled scikit-learn model. You can pass the run ID and the path of the model
 within the artifacts directory (here "model") to various tools. For example, MLflow includes a
-simple REST server for scikit-learn models:
+simple REST server for python-based models:
 
 .. code:: bash
 
-    mlflow sklearn serve -r <RUN_ID> -m model
+    mlflow pyfunc serve -r <RUN_ID> -m model
 
 .. note::
 
     By default the server runs on port 5000. If that port is already in use, use the `--port` option to
-    specify a different port. For example: ``mlflow sklearn serve --port 1234 -r <RUN_ID> -m model``
+    specify a different port. For example: ``mlflow pyfunc serve --port 1234 -r <RUN_ID> -m model``
 
-Once you have started the server, you can pass it some sample data with ``curl`` and see the
-predictions:
+Once you have started the server, you can pass it some sample data and see the
+predictions. The server accepts the following data formats as inputs:
+
+    - JSON-serialized Pandas Dataframes in the `split` orientation.
```

Comment:
nit: Pandas DataFrames

Reply:
Done (replaced this throughout the updated docs)